dev/sbt-scripted-library/src/main/java/impl/FooModule.java

```diff
@@ -29,11 +29,11 @@ public FooOnStart(Application app) {
     private void doOnStart(Application app) {
         try {
             // open for append
-            FileWriter writer = new FileWriter(app.getFile("target/reload.log"), true);
+            FileWriter writer = new FileWriter(app.getWrappedApplication().environment().getFile("target/reload.log"), true);
```

Comment:
Why not inject `play.Environment` and avoid `app.getWrappedApplication().environment()` here?

Reply:
Small PR I've made so that, if you need to inject an `Application`, you can directly use `app.environment`.